ts/screens/preferences/OldServicesHomeScreen.tsx

```diff
@@ -124,25 +125,33 @@ class OldServicesHomeScreen extends React.Component<Props> {
   };
 }
 
+const servicesSelector = (state: GlobalState) => state.entities.services;
+const organizationsSelector = (state: GlobalState) =>
+  state.entities.organizations;
+
+export const getAllSections = createSelector(
+  [servicesSelector, organizationsSelector],
+  (services, organizations) => {
+    const orgfiscalCodes = Object.keys(services.byOrgFiscalCode);
+    return orgfiscalCodes
+      .map(fiscalCode => {
+        const title = organizations[fiscalCode] || fiscalCode;
+        const serviceIdsForOrg = services.byOrgFiscalCode[fiscalCode] || [];
+        const data = serviceIdsForOrg
+          .map(id => services.byId[id])
+          .filter(isDefined);
+        return {
+          title,
+          data
+        };
+      })
+      .sort((a, b) => (a.title || "").localeCompare(b.title))
```

Comment:
@Alex193a  You could sort the array just after filtered it, just an optimization ;)

Reply:
Done! [0c97f3c](https://github.com/teamdigitale/io-app/pull/1153/commits/0c97f3ceeee8cb4bfd1d4f560c78ee1457efa7b2)